AutoCollection/CorrelationContextManager.ts

```diff
@@ -299,3 +250,53 @@ export class CorrelationContextManager {
         global.Error = <any>AppInsightsAsyncCorrelatedErrorWrapper;
     }
 }
+
+class CustomPropertiesImpl implements CustomProperties {
```

Comment:
This should implement `PrivateCustomProperties`, or that interface should be removed in favor of this class (which would require some minor changes in the other classes which reference it currently)